rocAL/rocAL_pybind/examples/rocAL_api_tf_classification_reader.py

```diff
@@ -30,11 +30,13 @@
 
 def draw_patches(img, idx, device_type):
     import cv2
+    args = parse_args()
     if device_type == "gpu":
-        img= cp.asnumpy(img)
-    image = img.transpose([0,1,2])
-    image = cv2.cvtColor(image, cv2.COLOR_RGB2BGR)
-    cv2.imwrite("OUTPUT_IMAGES_PYTHON/NEW_API/TF_READER/CLASSIFICATION/" + str(idx)+"_"+"train"+".png", image)
+        img = cp.asnumpy(img)
+    if not args.NHWC:
```

Comment:
I know it is used in many examples. But we need to output the same format needed by the application as mentioned which will also get rid of dependancy on args

Reply:
We added the option for the user to test both output layouts for the augmentation so we are transposing the images to HWC format here for dumping and visualizing the augmentation outputs.